fedimint-core/src/envs.rs

```diff
@@ -37,7 +33,7 @@ pub fn is_rbf_withdrawal_enabled() -> bool {
     is_env_var_set("FM_UNSAFE_ENABLE_RBF_WITHDRAWAL")
 }
 
-/// Get value of [`FEDIMINT_BUILD_CODE_VERSION_ENV`] at compile time
+/// Get value of `FEDIMINT_BUILD_CODE_VERSION` at compile time
```

Comment:
Note for reviewer: We have to remove the brackets since we removed the rust const it's referencing. Rather than reference that rust const, now we just reference the environment variable.